### ISSUE TITLE
Enable Wikis

### ISSUE THREAD

**Issue by f1shy-dev** [benawad/dogehouse#1365]
**Is your feature request related to a problem? Please describe.**
I think we should enable Wikis in this project, so we can make easy to follow guidelines for contributing/editing. For example, multiple people frequently ask how to add languages, so a Wiki page about that would make it simpler!

**Describe the solution you'd like**
@benawad needs to enable Wikis in this repo, and also make it so people can contribute to them.

**Describe alternatives you've considered**
We could just make `.md` files with all of the information (we kinda do this already), but that's not really a good solution if there is already a built-in wikis feature.

**Comment by HarrisonMayotte** [comment]
I agree and disagree at the same time. I think there are benefits to both approaches and I think it would make more sense for now to have the .md files, but I'm not opposed to the Wiki either. I just think that the wiki has to be maintained by collaborators rather than just any contributor due to the way github works.

**Comment by f1shy-dev** [comment]
Yeah I guess, not really sure what to do, I guess we wait for Ben's view on this.

**Comment by vi241** [comment]
I think Ben needs to do this

**Comment by HonbraDev** [comment]
Based on the conversation, I think we should have MarkDown files because anyone can contribute.

**Comment by vi241** [comment]
@HonbraDev markdown is one word

**Comment by mmattbtw** [comment]
We could do a [GitBook](https://gitbook.com), which has GitHub integration and uses markdown, so you can sync your GitBook wiki to a GitHub repo, and people can contribute on GitHub, which will automatically upload to the GitBook wiki.

**Comment by vi241** [comment]
I think wikis are good or just plain markdown

**Comment by HarrisonMayotte** [comment]
I would say GitBook would make sense just because it would allow more people than just collaborators to work on it but it would still be controlled because the changed would have to go through the same PR process.

**Comment by HarrisonMayotte** [comment]
It's up to @benawad though.

**Comment by vi241** [comment]
yes

**Comment by overlisted** [comment]
I can enable them too

**Comment by HarrisonMayotte** [comment]
All collaborators can enable them but it's about whether or not that's the commitment we want to make because it's either all in on wikis, all in on Gitbook, or all in on whatever other thing we decide to use for the information management.

**Comment by vi241** [comment]
how can we?

**Comment by overlisted** [comment]
just open the wiki tab

**Comment by TheOtterlord** [comment]
I think embeding the docs in the codebase as markdown files would work better, as then anyone can edit it, but it is all moderated by the PR system

**Comment by f1shy-dev** [comment]
GitBook could work, but whatever we do it's still up-to @benawad.

**Comment by HonbraDev** [comment]
@ofsho autocorrect

**Comment by vi241** [comment]
@HonbraDev what?

**Comment by HonbraDev** [comment]
I replied to a notofication on my phone, it seems that you deleted the message. It was about MarkDown being one word.

**Comment by vi241** [comment]
oh ok

**Comment by imadill** [comment]
So... Did Ben said anything about this matter? I was thinking to open a issue to close the Wiki tab because that was unused until I found this issue.

**Comment by TheOtterlord** [comment]
He's not commented on this yet no. I doubt we'll end up using wikis as then only collaborators can edit it. For anyone wishing to contribute documentation, do so in `./docs` for now
